salt/journal/config/srv-journal-app-config-parameters.yml

```diff
@@ -1,12 +1,14 @@
-{% set on_elasticache = salt['elife.cfg']('cfn.outputs.ElastiCacheHost') %}
 {% set standard_keys = ['secret', 'api_key', 'api_url', 'api_url_public', 'side_by_side_view_url', 'session_name', 'status_checks', 'gtm_id', 'disqus_domain', 'oauth2_client_id', 'oauth2_client_secret', 'hypothesis_api', 'hypothesis_authority', 'hypothesis_client_id', 'hypothesis_client_secret', 'feature_can_view_webp_countries']  %}
 
 parameters:
     {% if pillar.journal.get('redis') == false %}
     redis: null
-    {% elif on_elasticache %}
+    {% elif salt['elife.cfg']('cfn.outputs.ElastiCacheHost') %}
     {% set remote_redis = "redis://" + salt['elife.cfg']('cfn.outputs.ElastiCacheHost') + ":" + salt['elife.cfg']('cfn.outputs.ElastiCachePort') %}
     redis: {{ remote_redis }}
+    {% elif salt['elife.cfg']('cfn.outputs.ElastiCacheHost1') %}
+    {% set remote_redis = "redis://" + salt['elife.cfg']('cfn.outputs.ElastiCacheHost1') + ":" + salt['elife.cfg']('cfn.outputs.ElastiCachePort1') %}
+    redis: {{ remote_redis }}
```

Comment:
The variables are a bit redundant?

Reply:
Will simplify when removing backward compatibility.